docs/source/en/index.mdx

```diff
@@ -161,16 +160,13 @@ The library currently contains JAX, PyTorch and TensorFlow implementations, pret
 1. **[XLSR-Wav2Vec2](model_doc/xlsr_wav2vec2)** (from Facebook AI) released with the paper [Unsupervised Cross-Lingual Representation Learning For Speech Recognition](https://arxiv.org/abs/2006.13979) by Alexis Conneau, Alexei Baevski, Ronan Collobert, Abdelrahman Mohamed, Michael Auli.
 1. **[XLS-R](model_doc/xls_r)** (from Facebook AI) released with the paper [XLS-R: Self-supervised Cross-lingual Speech Representation Learning at Scale](https://arxiv.org/abs/2111.09296) by Arun Babu, Changhan Wang, Andros Tjandra, Kushal Lakhotia, Qiantong Xu, Naman Goyal, Kritika Singh, Patrick von Platen, Yatharth Saraf, Juan Pino, Alexei Baevski, Alexis Conneau, Michael Auli.
 1. **[YOSO](model_doc/yoso)** (from the University of Wisconsin - Madison) released with the paper [You Only Sample (Almost) Once: Linear Cost Self-Attention Via Bernoulli Sampling](https://arxiv.org/abs/2111.09714) by Zhanpeng Zeng, Yunyang Xiong, Sathya N. Ravi, Shailesh Acharya, Glenn Fung, Vikas Singh.
-
-
```

Comment:
I saw multiple blank lines that have been removed, do you know why?

Reply:
I just ran, the `make style`, `make fixup`, `make quality` commands for code reformatting, I think these blank lines got removed in running these commands as I didn't made any changes manually. If needed I can revert these changes which removed the blank lines.

Reply:
> I just ran, the `make style`, `make fixup`, `make quality` commands for code reformatting, 

Ok it seems to be fine. I usually only use `make fixup` because this command do everything (style + quality).

Let's see what Transformers maintainers think about this. @lewtun

Reply:
I have reverted the formatting changes and rebased with `main`  branch.